cmd/truss/main.go

```diff
@@ -39,7 +64,7 @@ func init() {
 
 	if len(flag.Args()) == 0 {
 		fmt.Fprintf(os.Stderr, "%s: missing .proto file(s)\n", filepath.Base(os.Args[0]))
-		fmt.Fprintf(os.Stderr, "Try '%s --help' for more information.\n", filepath.Base(os.Args[0]))
+		flag.Usage()
```

Comment:
This was originally done to mirror the output of common command line tools `cp` and `mv` when you call them with no operands.
cp:
```
$ cp
cp: missing file operand
Try 'cp --help' for more information.
```
truss:
```
$ truss
truss: missing .proto file(s)
Try 'truss --help' for more information.
```

Now I feel it is a bit verbose when you don't pass anything
```
$ truss
truss: missing .proto file(s)
Truss (version: 9f6da34c52 built: 2016-12-06 22:49:08 UTC)
Usage: truss [OPTION]... [*.proto]...
  -pbout string
    	The go package path where the protoc-gen-go .pb.go structs will be written.
  -svcout string
    	The go package path where the generated service directory will be written.
  -v	Verbose output with stack traces for errors.
```

What is your reasoning here?